client/src/plus/offer-overview/offer-overview-subscribe/index.tsx

```diff
@@ -179,7 +182,13 @@ function OfferDetails({
         )}
         {(current && <span className="sub-link current">Current plan</span>) ||
           (upgrade === null && (
-            <a href={ctaLink} className="sub-link">
+            <a
+              href={ctaLink}
+              className="sub-link"
+              onClick={() =>
+                gleanClick(`${OFFER_OVERVIEW_CLICK}_${offerDetails.id}`)
```

Comment:
I _think_ the de-facto convention we use is this:

```suggestion
                gleanClick(`${OFFER_OVERVIEW_CLICK}: ${offerDetails.id}`)
```

This makes it easier to distinguish the metrics name from its key/param.